README.md

```diff
@@ -1,3 +1,66 @@
 # apostrophe-seo
 
-Test.
+SEO for [ApostropheCMS](http://apostrophecms.org/).
+
+Add useful meta fields to `apostrophe-custom-pages`, `apostrophe-pieces`, and `apostrophe-global`.
+
+## Installation
+
+```bash
+npm install apostrophe-seo --save
+```
+
+## Use
+
+Configure `apostrophe-seo` in `app.js`.
+
+```js
+const apos = require('apostrophe')({
+  shortName: 'project',
+  modules: {
+    'apostrophe-seo': {}
+  }
+```
+
+Add to `outerLayout.html` in `apostrophe-templates/views/`.
```

Comment:
Add the following to ... or create it. Make sure it extends outerLayoutBase...

Reply:
I'm saying that other people might not have an outerLayout.html yet at all. They could actually just do all this in `layout.html`. So it might be easier to explain it as "add the following to the `layout.html` that all of your pages extend, or to `outerLayout.html` if you have one."